aztk/models/plugins/internal/plugin_manager.py

```diff
@@ -26,6 +26,9 @@ class PluginManager:
         tensorflow_on_spark=plugins.TensorflowOnSparkPlugin,
         openblas=plugins.OpenBLASPlugin,
         nvblas=plugins.NvBLASPlugin,
+        pip=plugins.PipPlugin,
+        apt_get=plugins.AptGetPlugin,
```

Comment:
should we name this one "apt-get" or just "apt"?

Reply:
technically, since we are calling `apt-get` and not `apt` I think this is fine as is.

Reply:
I think we should call these `apt_get_install`, `conda_install` and `pip_install` since that is more indicative of what the plugin does.

Reply:
Though to compare, this is travis apt plugin https://docs.travis-ci.com/user/installing-dependencies/#Adding-APT-Packages

Reply:
This also has an option to update or not before

Reply:
I can't think of a case where you wouldn't want to update.

Reply:
Going to stick with the current convention of apt_get since the rest of the plugins are with '_'